Program.cs

```diff
@@ -1,16 +1,20 @@
 ﻿using System;
 for (; ; ) {
    Console.Write ("Enter a decimal number: ");
-   if (int.TryParse (Console.ReadLine (), out int decimalnum) && decimalnum>0) {
-      Console.WriteLine ($"Hexadecimal:{decimalnum:X}");
-      string binary = "";
-      while (decimalnum > 0) {
-         int remainder = decimalnum % 2;
-         binary = remainder.ToString () + binary;
-         decimalnum /= 2;
+   if (int.TryParse (Console.ReadLine (), out int decimalnum)) {
+      if (decimalnum >= 0) {
+         Console.WriteLine ($"Hexadecimal: {decimalnum:X}");

```

Comment:
Please write the Hexadecimal conversion like binary.

Reply:
Done Sir